src/Categories/Category/Construction/Properties/CoEilenbergMoore.agda

```diff
@@ -40,29 +41,43 @@ module _ {F : Functor 𝒟 𝒞} {G : Functor 𝒞 𝒟} (F⊣G : Adjoint F G) w
     module T = Comonad T
     module F = Functor F
     module G = Functor G
+    module FG = Functor (F ∘F G)
 
     open Adjoint F⊣G
     open NaturalTransformation
+    open Category.Equiv
 
   -- Maclane's Comparison Functor
   ComparisonF : Functor 𝒟 coEM𝒞
   ComparisonF = record
    { F₀ = λ X → record
     { A = F.F₀ X
     ; coaction = F.F₁ (unit.η X)
-    ; commute = {!   !}
+    ; commute = commute-obj
     ; identity = zig
     }
    ; F₁ = λ {A} {B} f → record
     { arr = F.F₁ f
-    ; commute = begin
-      F.F₁ (unit.η B) 𝒞.∘ F.F₁ f ≈⟨ {!   !} ⟩
-      T.F.F₁ (F.F₁ f) 𝒞.∘ F.F₁ (unit.η A) ∎
+    ; commute = commute-mor
     }
    ; identity = F.identity
    ; homomorphism = F.homomorphism
    ; F-resp-≈ = F.F-resp-≈
    }
+   where
+    commute-obj : {X : Category.Obj 𝒟} → T.F.F₁ (F.F₁ (unit.η X)) 𝒞.∘ F.F₁ (unit.η X) 𝒞.≈ T.δ.η (F.F₀ X) 𝒞.∘ F.F₁ (unit.η X)
+    commute-obj {X} = begin
+      F.F₁ (G.F₁ (F.F₁ (unit.η X))) 𝒞.∘ F.F₁ (unit.η X) ≈⟨ sym 𝒞 (Functor.homomorphism F) ⟩
```

Comment:
you have `module F` defined, so you can use `F.homomorphism` instead of `Functor.homomophism F`.